### PR TITLE
Revised `Test Cases`.

### PR DESCRIPTION
**Pull request by m1cm1c**
More is coming but I want to get this through.

### CODE REVIEW

**Comment by RomanLangrehr** [comment]
Lgtm

**Comment by flyfl** [comment]
lgtm

File: doc/Requirements Specification/Parts/Test Cases.lyx [hunk 1]
```diff
@@ -536,8 +536,8 @@ gls{code decorator}
 
 \end_inset
 
- does not fit to the code) and does not crash but responds to it in a acceptable
- way.
+ does not fit to the code) and does not crash but responds to it in an acceptabl
+e way.
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: There is a space: `acceptabl e`

File: doc/Requirements Specification/Parts/Test Cases.lyx [hunk 2]
```diff
@@ -230,8 +230,8 @@ gls{Beagle}
 \end_inset
 
  discovers all sections needed for measurement and that they are correct.
- A part of this can be realised by checking if all code sections (of the
- measured part) were measured.
+ A part of this can be implemented by checking if all code sections (of
+ the measured part) have been measured.
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: “ have been executed”!!!

**Comment by jGleitz** [comment]
lgtm if the above is fixed. If the next commit only fixes the above sentence, this may be merged without waiting for Travis.

File: doc/Requirements Specification/Parts/Test Cases.lyx (hunk 1 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: Not in the pdf.

Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: Neither in Lyx.

**Pull request merged**